### PR TITLE
docs: pinned footer to bottom of page regardless of page height

### PR DESCRIPTION
**Pull request by Kamran64**
## Checklist

- [x] commit message follows [commit guidelines](https://github.com/appsody/website/blob/master/CONTRIBUTING.md#commit-message-guidelines).
- [x] website builds and runs in production - for information on how to test locally go [here](https://github.com/appsody/website/blob/master/DEVELOPMENT.md).

## Description
Reintroduced a fix to pin the footer to the bottom of the page for short docs.

## Related Issues
Fixes: #161

### CODE REVIEW

File: src/styles/main.css [hunk 1]
```diff
@@ -385,6 +391,7 @@ footer a:hover {
 
 .doc-content {
   min-height: calc(100vh - 10.5em);
```
Comment (tomleah) [inline src/styles/main.css]: Can you verify if we need this line anymore? I think it was from the previous fix

File: src/styles/main.css [hunk 2]
```diff
@@ -444,3 +451,7 @@ footer a:hover {
 .hide {
   display: none;
 }
+
+html {
```
Comment (tomleah) [inline src/styles/main.css]: nit: could you move this to the top of the file above `body`

File: src/styles/main.css (hunk 1 above)
Comment (Kamran64) [reply src/styles/main.css]: You're right, it's been removed :)

File: src/styles/main.css (hunk 2 above)
Comment (Kamran64) [reply src/styles/main.css]: Done

**Pull request merged**